dups/utils.py

```diff
@@ -52,7 +52,12 @@ def notify(title, body=None, icon=None, app_name=None):
     noti = Notify.Notification.new(title, body, icon)
     if app_name:
         noti.set_app_name(app_name)
-    noti.show()
+    try:
+        noti.show()
+    except Exception as e:
```

Comment:
Needs two more changes:
 1. remove the `as e` (unused variable)
 2. put a `pass` after the comment

Reply:
Done